_posts/en/newsletters/2019-05-21-newsletter.md

```diff
@@ -0,0 +1,221 @@
+---
+title: 'Bitcoin Optech Newsletter #47'
+permalink: /en/newsletters/2019/05/21/
+name: 2019-05-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the bip-anyprevout soft fork proposal, summarizes
+a few technical talks from the Magical Crypto Friends conference, and
+includes our regular sections on bech32 sending support and notable
+changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposed anyprevout sighash modes:** two weeks ago, Anthony Towns
+  posted a proposed BIP to the Bitcoin-Dev and Lightning-Dev mailing
+  lists for consideration.  The idea, [bip-anyprevout][], provides the
+  functionality previously described by [BIP118][] with modifications to
+  work with Taproot and to reduce the risk of accidental misuse.  One of
+  the signature hashing (sighash) modes available with bip-anyprevout is
+  directly compatible with the proposed [Eltoo][] layer for LN,
+  requiring only modifications for Taproot and the addition of a
+  *chaperone signature* (described later).  A second sighash mode
+  commits to more data than necessary for Eltoo in a way that may make
+  it useful for atypical commitments in Eltoo or for use in other protocols.
+
+    A significant advantage of this proposal over BIP118 noinput is that
+    it can make use of Taproot cooperative spends, allowing the two or
+    more parties to an LN channel or other contract protocol to spend
+    their funds without revealing any of the contract terms (including
+    that anyprevout was in use).  However, if they do need to reveal the
```

Comment:
I think this sentence "However, if they do need to reveal the..." is unnecessary, given the thorough treatment of taproot that you gave last week, and would be slightly in favour of removing it.